File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -1834,3 +1834,171 @@ document.Close(True)
 {% endhighlight %}
 
 {% endtabs %}
+
+## Inject Custom CSS
+
+The Blink rendering engine provides the support for inject the custom CSS to be applied in HTML or URL before rendering to PDF. 
```

Comment:
Use this - "The Blink rendering engine supports the injection of custom CSS to be applied to HTML or a URL before rendering it into a PDF document using the 'Css' property of the 'BlinkConverterSettings' class."

Reply:
Provide API link for BlinkConverterSettings class